css-gaps-1/Overview.bs

```diff
@@ -975,6 +975,162 @@ Lists of values and the ''repeat-line-color/repeat()'' notation</h3>
+	</ol>
+	<div class="example">
+		<pre class="lang-css">
+		@keyframes example {
+			from { column-rule-width: 10px; }
+			to   { column-rule-width: 20px 40px; }
+		}
+		</pre>
+		Interpolation of the above values would result in expansion of the
+		"from" and "to" values to create lists of equal lengths:
+		<pre>
+		From:   10px 10px
+		At 50%: 15px 25px
+		To:     20px 40px
+		</pre>
+	</div>
+	<div class="example">
+		<pre class="lang-css">
+		@keyframes example {
+			from { column-rule-width: repeat(2, 5px 10px); }
+			to   { column-rule-width: repeat(2, 15px 20px); }
+		}
+		</pre>
+		Interpolation of the above values would result in expansion of the
+		"from" and "to" values to create lists of equal lengths:
+		<pre>
+		From:    5px 10px  5px 10px
+		At 50%: 10px 15px 10px 15px
+		To:     15px 20px 15px 20px
+		</pre>
+	</div>
+
+	<div class="example">
+		<pre class="lang-css">
+		@keyframes example {
+			from { column-rule-width: repeat(2, 10px 20px); }
+			to   { column-rule-width: 20px; }
+		}
+		</pre>
+		Interpolation of the above values would result in expansion of the
+		"from" and "to" values to create lists of equal lengths:
+		<pre>
+		From:   10px 20px 10px 20px
+		At 50%: 15px 20px 15px 20px
+		To:     20px 20px 20px 20px
+		</pre>
+	</div>
+	<div class="example">
+		<pre class="lang-css">
+		@keyframes example {
+			from { column-rule-width: repeat(2, 10px 20px); }
+			to   { column-rule-width: 20px 30px; }
+		}
+		</pre>
+		Interpolation of the above values would result in expansion of the
+		"from" and "to" values to create lists of equal lengths:
+		<pre>
+		From:   10px 20px 10px 20px
+		At 50%: 15px 25px 15px 25px
+		To:     20px 30px 20px 30px
+		</pre>
+	</div>
+
+
+<h5 id="auto-repeaters">Lists with Auto Repeaters</h5>
+	<div algorithm>
+
+		To <dfn>interpolate with auto repeaters</dfn>, when either of the lists we are interpolating between (|from| and |to|) include an <a>auto repeater</a>:
+
+			<ol>
+				<li>
+				Split each of |from| and |to| into segments, similar to how we [=assign gap decoration values=]:
+				Let |leading values| be the values before the <a>auto repeater</a>.
+				Let |trailing values| be the values after the <a>auto repeater</a>.
+				Let |auto values| be the values inside the <a>auto repeater</a>.
+				</li>
+				<li>If either of |leading values| or |trailing values| in |from| or |to| is empty and the other has an <a>auto repeater</a>, we fall back to [=discrete=] interpolation.</li>
+				<li>Expand any integer repeaters on each segment.</li>
+				<li>
+				If the length of |leading values| in |from| and |leading values| in |to| don't match, we fall back to [=discrete=] interpolation.
+				If the length of |trailing values| in |from| and |trailing values| in |to| don't match, we fall back to [=discrete=] interpolation.
+				</li>
+				<li>If only one of |from| or |to| contains an auto-repeater, we fall back to [=discrete=] interpolation.</li>
```

Comment:
Wondering if it would make sense to check this condition first, before we analyze leading/trailing values in depth? I think that would also eliminate the need to check for an auto repeater in step 2.